### PR TITLE
[dom daint] Fix Boost 1.65.0 with python3 support

### PR DESCRIPTION
**Pull request by teojgo**

### CODE REVIEW

File: easybuild/easyconfigs/b/Boost/Boost-1.65.0-CrayGNU-17.08-python3.eb [hunk 1]
```diff
@@ -13,15 +13,21 @@ toolchainopts = {'pic': True, 'usempi': True, 'verbose': False}
 sources = ['%%(namelower)s_%s.tar.gz' % '_'.join(version.split('.'))]
 source_urls = ['https://sourceforge.net/projects/%(namelower)s/files/%(namelower)s/%(version)s']
 
-configopts  = ' --with-python=$EBROOTCRAYPYTHON/bin/python3 --with-python-version=3.5'
-configopts += ' --with-python-root=$EBROOTCRAYPYTHON/lib/python3.5'
+configopts  = ' --with-python=/opt/python/17.06.1/bin/python3'
```
Comment (jgphpc) [inline easybuild/easyconfigs/b/Boost/Boost-1.65.0-CrayGNU-17.08-python3.eb]: patch looks good. 
Can you use 
`--with-python=$EBROOTCRAYPYTHON/bin/python3` ?

Comment (teojgo) [reply easybuild/easyconfigs/b/Boost/Boost-1.65.0-CrayGNU-17.08-python3.eb]: Yeap no problem

**Comment by teojgo** [comment]
retest this please

**Comment by teojgo** [comment]
retest this please

**Comment by teojgo** [comment]
retest this please

**Comment by teojgo** [comment]
retest this

**Comment by teojgo** [comment]
retest this please

**Comment by teojgo** [comment]
retest this please

**Comment by jgphpc** [comment]
If you want this in production, edit and push 
https://github.com/eth-cscs/production/blob/master/jenkins-builds/6.0.UP04-17.08-gpu
and
https://github.com/eth-cscs/production/blob/master/jenkins-builds/6.0.UP04-17.08-mc

**Pull request merged**